### PR TITLE
Update the footer.

### PR DESCRIPTION
**Pull request by mldk-tech**
#64

### CODE REVIEW

**Comment by mldk-tech** [comment]
Update IBLI footer.

ibli_footer

**Comment by mldk-tech** [comment]
@bricel 

Put the footer into a mini panel?

File: ibli/themes/custom/bootstrap_subtheme/templates/page.tpl.php [hunk 1]
```diff
@@ -225,7 +236,7 @@
       <div class="col-sm-4">
         <div class="headline"><h3>Subscribe</h3></div>
         <div class="content">
-          <p>Enter your e-mail below to subscribe to our free newsletter.<br />We promise not to bother you often!</p>
+          <p>Enter your email or mobile phone number to receive updates about IBLI" with a text box,<br /> currently disconnected.</p>
```
Comment (NofarGizra) [inline ibli/themes/custom/bootstrap_subtheme/templates/page.tpl.php]: Please remove the part after the `"` , it was part of the issue, not the text...

**Comment by NofarGizra** [comment]
Merged.

**Pull request closed without merging**